creator-node/compose/env/base.env

```diff
@@ -27,29 +40,17 @@ delegatePrivateKey=
 dataProviderUrl=http://audius_ganache_cli:8545
 spOwnerWalletIndex=
 
-# Run as creator node by default
-isUserMetadataNode=false
-
-# Put CNs to debug mode if we're running locally.
-# Can be overriden.
-creatorNodeIsDebug=true
 
-WAIT_HOSTS=
-enableRehydrate=true
-
-# peerSetManager configs
-peerHealthCheckRequestTimeout=2000
-# bytes; 10gb
-minimumStoragePathSize=10000000000
-# bytes; 2gb 
-minimumMemoryAvailable=2000000000
-maxFileDescriptorsAllocatedPercentage=95
+# Sync / SnapbackSM configs
+snapbackReconfigEnabled=true
+snapbackJobInterval=10000 # ms
+snapbackModuloBase=2
```

Comment:
all re-ordering, except for the new `minimumSecondaryUserSyncSuccessPercent` on line 51